notifications_utils/version.py

```diff
@@ -1 +1 @@
-__version__ = '46.1.0'
+__version__ = '47.0.0'
```

Comment:
Although this is a breaking change to the ZendeskClient, nothing is using the code that was deleted.